src/include/sof/lib/dai-zephyr.h

```diff
@@ -0,0 +1,269 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2016 Intel Corporation. All rights reserved.
+ *
+ * Author: Liam Girdwood <liam.r.girdwood@linux.intel.com>
+ *         Keyon Jie <yang.jie@linux.intel.com>
+ */
+
+/**
+  * \file include/sof/lib/dai.h
+  * \brief DAI Drivers definition
+  * \author Liam Girdwood <liam.r.girdwood@linux.intel.com>
+  * \author Keyon Jie <yang.jie@linux.intel.com>
+  */
+
+#ifndef __SOF_LIB_DAI_ZEPHYR_H__
+#define __SOF_LIB_DAI_ZEPHYR_H__
+
+#include <platform/lib/dai.h>
+#include <sof/bit.h>
+#include <sof/list.h>
+#include <sof/lib/io.h>
+#include <sof/lib/memory.h>
+#include <sof/lib/dma.h>
+#include <sof/list.h>
+#include <sof/sof.h>
+#include <sof/spinlock.h>
+#include <sof/trace/trace.h>
+#include <sof/ipc/topology.h>
+#include <sof/audio/pcm_converter.h>
+#include <sof/audio/ipc-config.h>
+#include <errno.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <device.h>
+
+/** \brief If the device does not exist it will be created */
+#define DAI_CREAT		BIT(0)
+
+struct dai {
+	uint32_t index;
+	uint32_t type;
+	uint32_t dma_caps;
+	uint32_t dma_dev;
+	const struct device *drv;
```

Comment:
This structure (for Zephyr) should only contain members that are needed for Zephyr APIs (caps and dev are probably not needed).